mobile/lib/services/native_api.dart

```diff
@@ -240,6 +181,82 @@ class LocationService {
     return await Geolocator.getCurrentPosition();
   }
 
+  // Future<List<Address>> getLocalAddress(double lat, double lang) async {
```

Comment:
we could try to minimise comments in code, unless we are just temporary disabling some functionality or something along those lines.